platform/sampler/src/org/netbeans/modules/sampler/Sampler.java

```diff
@@ -187,6 +187,11 @@ public void run() {
         }, SAMPLER_RATE, SAMPLER_RATE);
     }
 
+    @SuppressWarnings("deprecation")
+    private static long findThreadId() {
```

Comment:
Another [future deprecation](https://github.com/apache/netbeans/pull/4795/files#r1161680325) - this time suppressed.